packages/apollo-voyager-metrics/src/index.ts

```diff
@@ -82,7 +82,7 @@ export function updateResolverMetrics (resolverInfo: any, responseTime: number)
 
 function applyMetricsMiddleware (app: Application, config: MetricsConfiguration) {
   const path = config && config.path ? config.path : '/metrics'
-  app.get(path, app)
+  app.get(path, getMetrics)
```

Comment:
haha, oh man...we should really have a test for this!!

Reply:
@wei-lee we will get it done see #58